packages/wonder-stuff-render-server/types/.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable import/no-commonjs */
 module.exports = {
-    extends: ["../.eslintrc.js"],
```

Comment:
This isn't necessary after getting rid of the `--config` option we were passing to `eslint`.